integration-tests/sceptre-project/config/14/A.yaml

```diff
@@ -0,0 +1,3 @@
+template:
```

Comment:
As noted above, I think you should make your stack group "drift" or something related to that, rather than `14`.